TaskManager/Modules/Home/Presenter/HomeViewPresenter.swift

```diff
@@ -24,5 +24,12 @@ class HomeViewPresenterImp: HomeViewPresenter {
     
     func viewDidLoad() {
         view?.showTask(tasks: dataStorage.tasks)
+        NotificationCenter.default.addObserver(self, selector: #selector(notificationRecieved), name: NSNotification.Name(rawValue: "task was added"), object: nil)
```

Comment:
Let's create an extension to NSNotification.Name and add a static let with this name, so we know we use the same name in all places

Reply:
Also let's give it a name with camel case or have some form of naming convention without spaces.